python/cudf_polars/tests/containers/test_dataframe.py

```diff
@@ -90,3 +92,39 @@ def test_shallow_copy():
     )
     assert df.columns[0].is_sorted == plc.types.Sorted.YES
     assert copy.columns[0].is_sorted == plc.types.Sorted.NO
+
+
+@pytest.mark.parametrize("nulls_last", [True, False])
+def test_flags_preserved(with_nulls, nulls_last):
```

Comment:
nit: I tend to include more information in the test name even if it's long, it seems to help triage issues better from the pytest summary where multiple tests may be failing from one central issue. Something like 

```suggestion
def test_sorted_and_null_order_flags_preserved(with_nulls, nulls_last):
```

Reply:
Sure, done.